packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-facet.tsx

```diff
@@ -182,7 +224,11 @@ export class AtomicRatingFacet
             isSelected={isSelected}
             i18n={this.bindings.i18n}
             onClick={onClick}
-          ></FacetValueLink>
+          >
+            <FacetValueLabelRating
+              icons={this.generateIconDisplay(facetValue)}
+            ></FacetValueLabelRating>
```

Comment:
Instead of taking icons as a parameter, I feel like it would be more useful if this functional component took a number of stars to display.

Additionally, this component could be useful for result templates, so giving it a more generic name (e.g., `StarRating`) and making it eventually display fractional numbers of stars (e.g., 4.5 stars) would be very useful.

Reply:
Renaming it to include `Icon` will be a better name. And actually, I like your idea of making this component handle the star display. I think it will simply things for the rating facet.